README.md

```diff
@@ -117,6 +125,7 @@ These values allow you to configure some of the things the generator does.
 * `postGenerationMessage`: If specified, this string will be printed after all other output, you can use it to give final information to the user, such as 'run `npm run start` to get started'
 * `defaultBranchName`: This string will be used as the branch name for the git initialization. Defaults to main `main`
 * `initialCommitMessage`: This string will be used as the commit message for the initial git commit. Defaults to `feat(generation): initial commit`
+* `doNotStoreResponses`: If this is set to `true`, create-using-template will NOT store any responses for your template.
```

Comment:
Should we make this opt-in instead of out for the sake of privacy?

Reply:
Good point, i'll reverse the logic.

Reply:
resolved in 26a819b549befc016bc6480f3e074b9597bc3a12